network_params.yaml

```diff
@@ -1,4 +1,11 @@
 optimism_package:
+  altda_deploy_config:
+    use_altda: true
+    da_commitment_type: KeccakCommitment
+    da_challenge_window: 100
+    da_resolve_window: 100
+    da_bond_size: 0
```

Comment:
TODO: remove all altda stuff from this file and move it to another network_params_altda.yaml example file